internal/profiler/profiler.go

```diff
@@ -21,3 +21,17 @@ func Init(svcName, version string, blockProfileRate int) {
 		}
 	}
 }
+
+// InitLightweight starts the supported profilers IFF the environment variable is set.
+// Compared to Init, it disables mutex profiling and forced GC to reduce its overhead.
```

Comment:
I actually don't know why we do AllocForceGC given the default is off. I wonder if it has a measurable impact on our perf. I suspect not. Then it makes sense. Why don't we use it for zoekt-git-index then?